arithmetic_analysis/jacobi_iteration_method.py

```diff
@@ -115,6 +115,20 @@ def jacobi_iteration_method(
 
     strictly_diagonally_dominant(table)
 
+    """
+    denom - a list of values along the diagonal
+    val - values of the last column of the table array
+
+    masks - boolean mask of all strings without diagonal
+    elements array coefficient_matrix
+
+    ttt - coefficient_matrix array values without diagonal elements
+    ind - column indexes for each row without diagonal elements
+    arr - list obtained by column indexes from the list init_val

```

Comment:
There shouldn't be such a long comment in the middle of the algorithm. Instead of writing a block comment explaining what each variable does, why not just make the variable names clearer? You could also just add comments to individual lines.

Reply:
Attached in the archived file 'test75.py.tar.gz', file 'test75.py'. It contains the 'jacobi_test' function, which calculates according to the new algorithm. The arrays 'coefficient', 'constant', 'init_val' and the number 'n' (ascending) are generated. In the first iteration 2 x 2, the last iteration we have an array of 256 x 256( 'coefficient').
To check the performance, the perfplot library is used, in which, in the 'equality_check' function, both received arrays are checked for equality.

The size of the 'coefficient' array is printed on each iteration:

After n > 256 the array becomes so much larger and np.inf and -np.inf appear, which leads to nan numbers, both in the new and old calculation.

Reply:
Corrected.
Double - At the very bottom, I wrote about performance tests with a large array (answered your message).